maze_transformer/training/tokenizer.py

```diff
@@ -60,3 +71,92 @@ def as_tokens(
             )
 
         return tokens
+
+
+class HuggingMazeTokenizer(PreTrainedTokenizer):
+    vocab: dict[str, int]  # map of token_ids to strings
+
+    bos_token: str = SPECIAL_TOKENS["start_path"]
```

Comment:
I'm torn on this one, because I also find them unclear, but they are consistent with [HuggingFace](https://huggingface.co/docs/transformers/main_classes/tokenizer#transformers.PreTrainedTokenizer)